buildlib/tools/test_mad.sh

```diff
@@ -48,6 +48,8 @@ docker_run_srv() {
         $IMAGE \
         bash -c "${PWD}/install/bin/ucx_perftest -K ${HCA} > \
             ${PWD}/ucx_perf_srv_${test_name}_${BUILD_BUILDID}.log 2>&1"
+
+    sudo chown -R swx-azure-svc:ecryptfs "$PWD"
```

Comment:
why docker creates files outside the container? maybe we can avoid it?

Reply:
if it is needed to be able to later retrieve logs, it could be possible to either:
- use `docker ... bash -c "umask 0011; ${PWD}/install/bin/ucx_perftest ... "`
- run docker without root user like `docker --user $(id -u swx-azure-svc):$(id -g swx-azure-svc) ...`.